themes/gruvbox-dark-medium.json

```diff
@@ -1,6 +1,62 @@
 {
   "name":"Gruvbox Dark Medium",
-  "settings":[
+  "type": "dark",
+  "colors" : {
+        "activityBar.background": "#282828",
+        "activityBar.foreground": "#ebdbb2",
+        "activityBarBadge.background": "#458588",
+        "activityBarBadge.foreground": "#ebdbb2",
+        "sideBar.background": "#282828",
+
+        "focusBorder": "#bdae93",
+        "foreground": "#fbf1c7",
+        "editor.foreground": "#fbf1c7",
+
+        "list.activeSelectionBackground": "#665c54",
+        "list.inactiveSelectionBackground": "#504945",
+
+        "statusBar.background": "#282828",
+        "statusBar.noFolderBackground": "#282828",
+        "statusBar.foreground": "#ebdbb2",
+        "editorGroupHeader.tabsBackground": "#282828",
+        "tab.activeForeground": "#ebdbb2",
+
+        "input.background": "#282828",
+        "input.border": "#504945",
+
+        "editorWidget.background": "#282828",
+
+        "editor.selectionHighlightBackground": "#665c54",
+        "editor.selectionBackground": "#504945",
+        "editor.wordHighlightStrongBackground": "#928374",
+
+        "editor.background": "#282828",
+
+        "peekViewEditor.background": "#282828",
+        "peekViewResult.background": "#282828",
+        "peekViewResult.selectionBackground": "#928374",
+        "peekView.border": "#928374",
+
+        "terminal.ansiBlack": "#161819",
+        "terminal.ansiBrightBlack": "#7f7061",
+        "terminal.ansiRed": "#f73028",
+        "terminal.ansiBrightRed": "#be0f17",
+        "terminal.ansiGreen": "#aab01e",
+        "terminal.ansiBrightGreen": "#868715",
+        "terminal.ansiYellow": "#f7b125",
+        "terminal.ansiBrightYellow": "#cc881a",
+        "terminal.ansiBlue": "#719586",
+        "terminal.ansiBrightBlue": "#377375",
+        "terminal.ansiMagenta": "#c77089",
+        "terminal.ansiBrightMagenta": "#a04b73",
+        "terminal.ansiCyan": "#7db669",
+        "terminal.ansiBrightCyan": "#578e57",
+        "terminal.ansiWhite": "#faefbb",
+        "terminal.ansiBrightWhite": "#e6d4a3",
+        "terminal.foreground": "#faefbb",
+        "terminal.background": "#282828"
+  },
+  "tokenColors":[
     {
       "settings": {
```

Comment:
I noticed with the `tokenColors` change, this global `settings` block isn't affecting anything anymore, could you please remove this block.

One thing missing from removing this block is `lineHighlight`, so could you also add `"editor.lineHighlightBackground": "#32302f"` to the new global colors setting.